src/phone-input/country-select-dropdown.js

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import React from 'react';
+import {List, AutoSizer} from 'react-virtualized';
+
+import Flag from './flag.js';
+import {SIZE} from './constants.js';
+import {
+  StyledCountrySelectDropdownContainer as DefaultContainer,
+  StyledCountrySelectDropdownListItem as DefaultListItem,
+  StyledCountrySelectDropdownFlagColumn as DefaultFlagColumn,
+  StyledCountrySelectDropdownNameColumn as DefaultNameColumn,
+  StyledCountrySelectDropdownDialcodeColumn as DefaultDialcodeColumn,
+} from './styled-components.js';
+import {getOverrides} from '../helpers/overrides.js';
+
+import type {CountrySelectDropdownPropsT} from './types.js';
+
+export default function CountrySelectDropdown(
+  props: CountrySelectDropdownPropsT,
+) {
+  const {
+    children,
+    dropdownHeight = '400px',
+    mapIsoToLabel,
+    overrides = {
+      CountrySelectDropdown: {},
+      CountrySelectDropdownListItem: {},
+      CountrySelectDropdownFlagColumn: {},
+      CountrySelectDropdownNameColumn: {},
+      CountrySelectDropdownDialcodeColumn: {},
+    },
+  } = props;
+  const [Container, containerProps] = getOverrides(
+    overrides.CountrySelectDropdown,
+    DefaultContainer,
+  );
+  const [ListItem, listItemProps] = getOverrides(
+    overrides.CountrySelectDropdownListItem,
+    DefaultListItem,
+  );
+  const [FlagColumn, flagColumnProps] = getOverrides(
+    overrides.CountrySelectDropdownFlagColumn,
+    DefaultFlagColumn,
+  );
+  const [NameColumn, nameColumnProps] = getOverrides(
+    overrides.CountrySelectDropdownNameColumn,
+    DefaultNameColumn,
+  );
+  const [Dialcode, dialcodeProps] = getOverrides(
+    overrides.CountrySelectDropdownDialcodeColumn,
+    DefaultDialcodeColumn,
+  );
+  return (
+    <Container $height={dropdownHeight} {...containerProps}>
```

Comment:
Follow this [example](https://github.com/uber-web/baseui/pull/1207/files#diff-d2689559620ff39ce2a75901e58a7526R13) for how to get ref forwarding working. Right now, I'm unable to navigate the country menu with arrow keys

Reply:
ah i think it is because i am using a `Select` and not a `StatefulSelect` - maybe i can switch those out.

Reply:
Could we use Select's `maxDropdownHeight` props instead of passing the dropdown hight on the Container? The `maxDropdownHeight` is passed on the Menu's `List` element.

Reply:
😕it seems like the container needs to have an explicit height set or nothing shows up.